content/en/os/1.13.x/concepts/api-driven/index.markdown

```diff
@@ -0,0 +1,91 @@
++++
+title = "API Driven"
+type = "docs"
+description = "Administration and setting manipulation with the Bottlerocket API" 
++++
+
+Bottlerocket is an API-driven operating system.
+This is a departure from general purpose Linux distributions where you install packages, configure services through individual configuration files, and use commands from the shell to perform administrative tasks.
+
+With the concept of variants, all installed software for any given image is a known quantity and an included API enables you to configure everything from a single interface.
+Additionally, administrative task like checking for updates and rebooting as well as executing commands on the host are all accomplished with API calls.
+
+## Accessing the API
+
+The Bottlerocket API is only accessible by containers running on the same host with specific SELinux labels.
+The API is a Unix domain socket accessible by a mountable volume; containers running on the host or orchestrated containerd instance can access the API as long as they have both the correct SELinux labels and mount the socket.
```

Comment:
nit: instance -> instances?

Reply:
Yup. And caught another issue in that sentence to. 🤦‍♂️